test/contestant_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require "./lib/game"
+
+class ContestantTest < Minitest::Test
+
+  def test_it_exists
+    alexander = Contestant.new({first_name: 'Alexander',
+                                last_name: 'Aigiades',
+                                age: 28,
+                                state_of_residence: 'CO',
+                                spending_money: 10})
+    assert_instance_of Contestant, alexander
+  end
+
+  def test_it_has_attributes
+    alexander = Contestant.new({first_name: 'Alexander',
+                                last_name: 'Aigiades',
+                                age: 28,
+                                state_of_residence: 'CO',
+                                spending_money: 10})
+    assert_equal "Alexander Aigiades", alexander.full_name
+    assert_equal 28, alexander.age
+    assert_equal 'CO', alexander.state_of_residence
+    assert_equal 10, alexander.spending_money
+    assert_equal false, alexander.out_of_state?
```

Comment:
the interaction pattern shows 2 assertions for `out_of_state?` (one for true and one for false) which you should have too, whenever you have a method that can return a boolean.  in your case, it would have caught that `out_of_state` is always false.